style.css

```diff
@@ -0,0 +1,7 @@
+.title {
+    font-family: PT Serif;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/html_abrmhah/pullRequest?prid=6280495)